### DISCUSSION TITLE
[General] Up to date dashboards for various OP metrics

### DISCUSSION THREAD

**Discussion by Padraic-O-Mhuiris** [ethereum-optimism/developers#516] [General]
Apart from this [discussion](https://github.com/ethereum-optimism/developers/discussions/33) a few months ago, have there been any improved examples of dashboards which are more up to date? I've been googling a lot and examples outside of this are hard to find

**Comment by opfocus** [comment]
https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-9609201

**Reply by Padraic-O-Mhuiris** [reply]
Thanks!
